js/api.md

```diff
@@ -895,6 +895,7 @@ For example, the below code shows how you would update the `CreateTodoMutation`
 An example of using the `buildMutation` helper to add an item to the cache:
 
 ```javascript
+import { buildMutation } from 'aws-appsync/lib/helpers/offline';
```

Comment:
@ffxsam hi, can you change this to be `import { buildMutation } from 'aws-appsync';` because it is exported by this line: https://github.com/awslabs/aws-mobile-appsync-sdk-js/blob/master/packages/aws-appsync/src/index.ts#L12

Reply:
Done!